app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return unless user.present?
+
+    return unless user.admin?
+
+    can :destroy, Post, user: user
+    can :destroy, Comment, user: user
+
+    # https://github.com/CanCanCommunity/cancancan/blob/develop/docs/define_check_abilities.md
```

Comment:
- [ ] Right now, if the user is the owner of a post/comment they will not be able to destroy it, please make sure to adjust the method so the owner will be able to delete the post/comment as well.